treeopt/treeOptData/DoeData.csv

```diff
@@ -0,0 +1,20 @@
+3.331328124366226717e+00,-4.045744843788394718e+00
```

Comment:
What is this? I have a hunch this does not belong into the source directory `treeopt`, but rather into `tests/data`?

Reply:
I had the Idea, that all the parameters and the system responses should be stored in a csv file, so that if necessary a previously carried out experiment can be loaded into treeOpt and used for further analysis. The files in the folder treeOptData represent these safe files.

Reply:
I will add the folder to the .gitignore, so that it is not pushed to the repository on every commit